### ISSUE TITLE
Scrolling via gesture causes severe lags

### ISSUE THREAD

**Issue by tjanson** [radiant-player/radiant-player-mac#511]
Scrolling with the usual two-finger touchpad gesture causes the view to freeze, then jump – sometimes in a few small “stuttering” jumps, never in a fluid motion. This lag often exceeds 1 second, I’d even say it’s often closer to 3 secs.

Here’s the funny part: I wanted to make a screen recording (using QuickTime) to give a clear demonstration, but during the [first recording, the motion was much, much more fluid](https://vimeo.com/158184724). I immediately did [another recording, and the problem was back](https://vimeo.com/158184723) (unfortunately there’s no indication of when I’m scrolling, but I’m moving the mouse in circles during the lag).
I have not been able to reproduce the smooth scrolling. (I tried: screen capturing again, resizing, full screen, restarting Radiant.)

This happens regardless of the view (“Listen Now”, “My Library”, etc.) and even on the left-hand sidebar. The version is 1.7.2 (currently the latest). I’ve seen #408, which sounds very similar.

**Comment by chrismou** [comment]
Do you still see the issue with the custom title bar switched off?

Preferences > Appearance > Untick "use custom title bar" (will require a restart)

**Comment by tjanson** [comment]
That was already unchecked. I just tried all combinations of the two appearance settings, no change.

Oh, for the record, this is a MacBook Pro (13" late 2013, 2.4 GHz i5, 8 GB Ram) running OS X 10.11.4 Beta.

**Comment by ntpeters** [comment]
I'm having the exact same problem on an early 2015 13" Macbook Pro (3.1Ghz, 16GB RAM) running OSX 10.11.3.  I've tried disabling pretty much every special setting in preferences, and still am seeing this.

**Comment by ntpeters** [comment]
Actually, I just went through every setting one by one again, and I think I've narrowed it down to the "Enable swipe gestures to go back and forward" setting.  After disabling this and restarting Radiant Player, the UI is much more responsive.  I wouldn't say it is perfectly smooth, but it is significantly better than before.

@tjanson do you happen to have this option enabled as well?  Do things work any better for you with it disabled?

**Comment by tjanson** [comment]
@ntpeters Yes! That does improve things, thanks! I’ll add that to the issue description.

**Comment by jessejoe** [comment]
Confirming I'm seeing this same issue. I just opened Radiant for the first time and was wondering why simply scrolling up/down was so jittery. Disabling the gestures (which were enabled by default) and restarting the app made scrolling smooth.
